integration_test.go

```diff
@@ -351,6 +352,49 @@ func (suite *IntegrationTestSuite) TestSimpleImportStatement() {
 	)
 }
 
+func (suite *IntegrationTestSuite) TestSimpleImportStatementBigFile() {
+	database := suite.openConnection(suite.createDefaultConfig())
+	ctx := context.Background()
+	schemaName := "TEST_SCHEMA_8"
+	tableName := "TEST_TABLE_HUGE"
+
+	file, err := os.Create("./testData/data_huge.csv")
+	suite.NoError(err, "open file should be working")
+	defer file.Close()
+
+	writer := csv.NewWriter(file)
+	writer.Comma = ','
+
+	exampleData := time.Now().Format(time.RFC3339)
+
+	// Generate file with around 3 MB
+	for i := 0; i < 20000; i++ {
+		err := writer.Write([]string{fmt.Sprint(i), exampleData, exampleData, exampleData, exampleData, exampleData, exampleData})
+		suite.NoError(err, "adding example data should be working")
+	}
+	writer.Flush()
```

Comment:
Extract creation of the temp file into a separate method, delete temp file after test